packages/store-ui/src/organisms/SearchBar/SearchBar.tsx

```diff
@@ -0,0 +1,47 @@
+import type { ChangeEvent } from 'react'
+import React, { useRef, useState } from 'react'
+
+import type { PopoverProps } from '../../atoms/Popover'
+import Popover from '../../atoms/Popover'
+import type { SearchInputProps } from '../../molecules/SearchInput'
+import SearchInput from '../../molecules/SearchInput'
+
+export type SearchBarProps = Pick<
+  SearchInputProps,
+  'onSubmit' | 'onChange' | 'placeholder' | 'testId'
```

Comment:
The test id should be the same type, but I don't think that should be coupled with the SearchInputProps testid type. What do you think about it?